src/com/rcv/ResultsWriter.java

```diff
@@ -469,7 +469,8 @@ private void addActionRowCandidates(
   // param: worksheet to which we will be adding rows and cells
   // param: totalActiveVotesPerRound map of round to votes active in that round
   // returns: the next (empty) row index
-  private int addHeaderRows(XSSFSheet worksheet, Map<Integer, BigDecimal> totalActiveVotesPerRound) {
+  private int addHeaderRows(XSSFSheet worksheet,
```

Comment:
to be consistent with elsewhere, put each param on its own line

Reply:
Done.